src/services/layerHelper.js

```diff
@@ -48,7 +48,7 @@ ngeo.LayerHelper.GROUP_KEY = 'groupName';
  * @param {string} sourceURL The source URL.
  * @param {string} sourceLayersName A dot separated names string.
  * @param {string=} opt_serverType Type of the server ("mapserver",
- *     "geoserver", qgisserver, …).
+ *     "geoserver", "qgis", …).
```

Comment:
it's qgis not qgisserver in ol3 :-)
https://github.com/openlayers/ol3/blob/master/src/ol/source/wmssource.js